sync/sync.go

```diff
@@ -278,8 +286,14 @@ Register:
 		return err
 	}
 
+	logrus.Infof("syncFiles from:%v to:%v", fromClient, toClient)
 	if !ok {
-		logrus.Infof("syncFiles from:%v to:%v", fromClient, toClient)
+		if err = t.syncFiles(fromClient, toClient, output.Disks); err != nil {
+			return err
+		}
+	} else {
+		// Only sync head metadata
+		output.Disks = output.Disks[:1]
```

Comment:
Why do we need to sync head meta when chain is same? 
isn't parent in both replicas be same?